pythonFiles/vscode_datascience_helpers/dataframes/vscodeDataFrame.py

```diff
@@ -69,7 +69,11 @@ def _VSCODE_getDataFrameInfo(df):
         columnNames = list(df)
 
     # Compute the index column. It may have been renamed
-    indexColumn = df.index.name if df.index.name else "index"
+    try:
+        indexColumn = df.index.name if df.index.name else "index"
```

Comment:
@joyceerhl Just a slight tweak from your suggested code. Default dataframes have an index.name of None, so we still need the if statement here to revert to default "index" in that case.

Reply:
FYI - Please do not change `df.index.name or "index"` is also possible.

Reply:
@DonJayamanne Apologies, didn't mean to ignore this. Just missed the comment when added. I can change this later.